docs/write_test/development.md

```diff
@@ -14,14 +14,15 @@ what to expect of it), as well as our design and development philosophy.
 
 ## Environment Setup
 
-Follow the [installation](install.md) steps to prepare the source code. Then
-follow the steps below to set up the corresponding development environment.
+Follow the [installation](../quick_start#installation) steps to prepare the
+source code. Then follow the steps below to set up the corresponding development
+environment.
 
 ### Visual Studio Code
 
 1. Click on the Python version at the bottom left of the editor's window and
    select the Python interpreter which Poetry just created. If you do not find
-   it, check [FAQ and troubleshooting](troubleshooting.md) for extra
+   it, check [FAQ and troubleshooting](../troubleshooting.md) for extra
```

Comment:
Good to see they are fixed. We don't know when next PR arrives, so we should make sure no broken for each PR. This kind of change can be separated, and merged as soon as possible.